.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,21 @@
 ## Summary
+<!-- A short, clear statement of the proposed changes. -->
 
-Summarize your PR. If it involves visual changes include a screenshot or gif.
+## How to test
+<!-- Include steps for testing your PR and screenshots (if applicable). -->
 
+## Risks
+<!--
+Provide a matrix OR statements of known risks and how they are mitigated.
```

Comment:
I don't think we need a matrix for every PR: a clear statement covering the table is also good and spares authors of needing to edit MD tables.